libs/features/pages/consultancy-page/src/lib/consultancy-hero-section/consultancy-hero-section.component.scss

```diff
@@ -0,0 +1,89 @@
+/* Layout Properties for larger screens */
+.consultancy-main {
+    top: 0px;
+    left: 0px;
+    width: 1920px;
+    height: 1082px;
+    /**UI Properties*/
+    background: #dbc2f5 0% 0% no-repeat padding-box;
+    opacity: 1;
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 1920px;
+  }
+  
+  .text-section {
+    left: 209px;
+    width: 839px;
+    height: 106px;
+    text-align: left;
+    font: normal normal medium 82px/110px DM Sans;
```

Comment:
Also here, for the font, use CSS variables specified in the elements/theming/_variables.scss file